ur_robot_driver/src/hardware_interface.cpp

```diff
@@ -533,18 +533,18 @@ void URPositionHardwareInterface::handleRobotProgramState(bool program_running)
 void URPositionHardwareInterface::initAsyncIO()
 {
   for (size_t i = 0; i < 18; ++i) {
-    standard_dig_out_bits_cmd_[i] = static_cast<double>(actual_dig_out_bits_[i]);
+    standard_dig_out_bits_cmd_[i] = -1;
```

Comment:
Please define ```-1``` as a constant with a meaningful name and use that instead

Reply:
Done